asset/js/ets_woo_qa_script.js

```diff
@@ -13,7 +13,7 @@ jQuery(document).ready(function(){
 			data: 'action=ets_post_qusetion_answer&' + submit + "&add_qustion_nonce=" + etsWooQaParams.add_qustion_nonce,
 			success: function(res) {   
 				jQuery('.ets-submit').prop('disabled', false);
-				jQuery("#ques-text-ar").val("");
+				jQuery(".ets-qa-textarea").val("");
```

Comment:
You know, this will blank all the question textareas on the page. No problem but I hope you were aware.